src/nvim/editor/arbmark.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Same function but names are not limited to one char
+ * There is also no ex_command, just viml functions
+ */
+
+#include "nvim/vim.h"
+#include "nvim/globals.h"
+#include "nvim/mark.h"
+#include "nvim/memory.h"
+#include "nvim/map.h"
+#include "nvim/lib/kvec.h"
+#include "nvim/editor/arbmark.h"
+
+# define FOR_ALL_ARBMARKS(buf) \
+  kbitr_t = itr;                                              \
+  arbmark_T = *arbmark;                                       \
+  for (;kb_itr_valid(&itr)                                    \
+       ;kb_itr_next(arbmark_T, buf->b_arbmarks_tree; &itr)){
+
+static arbmark_T *_find_pos(pos_t pos, bool FORWARD);
+static int _arbmark_create(buf_T *buf, cstr_t *name,  pos_T *pos);
+static int _arbmark_update(buf_T *buf, arbmark_T *arbmark, pos_T *pos);
+static int _arbmark_delete(buf_T *buf, cstr_t *name);
+static arbmark_T *get_arbmark(buf_T *buf, cstr_t *name);
+static int pos_lt(pos_T *pos, pos_T *pos2);
+static int pos_eq(pos_T *pos, pos_T *pos2);
```

Comment:
@justinmk He is missing `#include` with generated functions. And he is missing it because until `editor/` subdirectory is added to the list nothing is going to be generated.